src/app/donations/donations.js

```diff
@@ -0,0 +1,59 @@
+angular.module('prx.donations', ['prx.bus', 'angulartics',])
+.run(function (Bus, $analytics) {
+  var category = 'Outbound';
+
+  Bus.on('donate.outbound', function (account, url) {
+    $analytics.eventTrack('Donate', {
+      category: category,
+      label: 'Account-' + account.id.toString() + '-' + url,
+      // hitcallback: function () {
+      //   $window.location.href = url;
+      // }
+    });
+  });
+})
+.service('prxDonateURL', function () {
+  this.forAccount = function (account) {
+    var map = {
+      '45139': 'http://themoth.org/support?utm_source=PRX&utm_medium=Donate%20Button&utm_campaign=PRX%20-%20Moth%20-%20Donate',
+      '103576': 'http://thetruthpodcast.com/Support.html',
+      '551': 'http://99percentinvisible.org/donate/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '12248': 'http://loveandradio.org/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '123677': 'http://www.storycentral.org/donate/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '31': 'http://toe.prx.org/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '561': 'http://www.radiodiaries.org/donate/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '579': 'http://www.kitchensisters.org/support/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+      '179116': 'http://www.theheartradio.org/?utm_source=PRX&utm_medium=donate_button&utm_campaign=Radiotopia',
+    };
+
+    return map[account.id.toString()];
+  };
+})
+.directive('prxDonate', function(prxDonateURL, $analytics, $window, $timeout, Bus) {
+  return {
+    restrict: 'E',
+    replace: true,
+    template: '<div class="donate">Donate</div>',
```

Comment:
I have definitely written code that looks like that, and I think it would not be the end of the world if it stayed this way.
We also have things like this elsewhere:
https://github.com/PRX/www.prx.org/blob/df6b791c1b3cdb0bc19b8e797b59c98955b46ac7/src/app/drawer/drawer.js#L70

That said, like I take @chrisrhoden to be saying, I could see using ng-click here with a controller in the directive, that seems like a better way to accomplish the same thing that gets the benefits and consistency of using ng-click.  I would probably vote for that, but I don't feel strongly.